### ISSUE TITLE
hyprland-nvidia-git fails to compile, missing wlroot-nvidia.patch

### ISSUE THREAD

**Issue by elofland** [prasanthrangan/hyprdots#591]
I'm pretty new to Arch and Hyprland, so I'll try to give the necessary details:
Intel Laptop with Nvidia Quadro P2000 (NV30)

Installed Arch 2023-11-01

Running the install.sh script works fine until this step:
```
Aur (1) hyprland-nvidia-git-0.32.3.r64.g512a5973-1
:: Proceed to review? [Y/n]: y

:: Downloading PKGBUILDs...
 PKGBUILDs up to date
 nothing new to review
fetching devel info...
==> Making package: hyprland-nvidia-git 0.32.3.r64.g512a5973-1 (Fri 01 Dec 2023 08:42:13 AM CST)
==> Retrieving sources...
  -> Updating hyprland-nvidia-git git repo...
==> Validating source files with sha256sums...
    hyprland-nvidia-git ... Skipped
==> Making package: hyprland-nvidia-git 0.32.3.r64.g512a5973-1 (Fri 01 Dec 2023 08:42:15 AM CST)
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> Retrieving sources...
  -> Updating hyprland-nvidia-git git repo...
==> Validating source files with sha256sums...
    hyprland-nvidia-git ... Skipped
==> Removing existing $srcdir/ directory...
==> Extracting sources...
  -> Creating working copy of hyprland-nvidia-git git repo...
Cloning into 'hyprland-nvidia-git'...
done.
==> Starting prepare()...
Submodule 'subprojects/hyprland-protocols' (https://github.com/hyprwm/hyprland-protocols) registered for path 'subprojects/hyprland-protocols'
Submodule 'subprojects/tracy' (https://github.com/wolfpld/tracy) registered for path 'subprojects/tracy'
Submodule 'subprojects/udis86' (https://github.com/canihavesomecoffee/udis86) registered for path 'subprojects/udis86'
Submodule 'wlroots' (https://gitlab.freedesktop.org/wlroots/wlroots.git) registered for path 'subprojects/wlroots'
Cloning into '/home/elofland/.cache/paru/clone/hyprland-nvidia-git/src/hyprland-nvidia-git/subprojects/hyprland-protocols'...
Cloning into '/home/elofland/.cache/paru/clone/hyprland-nvidia-git/src/hyprland-nvidia-git/subprojects/tracy'...
Cloning into '/home/elofland/.cache/paru/clone/hyprland-nvidia-git/src/hyprland-nvidia-git/subprojects/udis86'...
Cloning into '/home/elofland/.cache/paru/clone/hyprland-nvidia-git/src/hyprland-nvidia-git/subprojects/wlroots'...
Submodule path 'subprojects/hyprland-protocols': checked out '0c2ce70625cb30aef199cb388f99e19a61a6ce03'
Submodule path 'subprojects/tracy': checked out '37aff70dfa50cf6307b3fee6074d627dc2929143'
Submodule path 'subprojects/udis86': checked out '5336633af70f3917760a6d441ff02d93477b0c86'
Submodule path 'subprojects/wlroots': checked out '5d639394f3e83b01596dcd166a44a9a1a2583350'
/home/elofland/.cache/paru/clone/hyprland-nvidia-git/PKGBUILD: line 76: /home/elofland/.cache/paru/clone/hyprland-nvidia-git/src/hyprland-nvidia-git/nix/patches/wlroots-nvidia.patch: No such file or directory
==> ERROR: A failure occurred in prepare().
    Aborting...
error: failed to build 'hyprland-nvidia-git-0.32.3.r64.g512a5973-1':
error: packages failed to build: hyprland-nvidia-git-0.32.3.r64.g512a5973-1
```

I'm clearly missing the wlroots-nvidia.patch file:
```
$ ls ~/.cache/paru/clone/hyprland-nvidia-git/src/hyprland-nvidia-git/nix/patches/
meson-build.patch
```

Not sure why this wasn't pulled, but any pointers would be greatly appreciated!

**Comment by Ben-Plagmann** [comment]
This is because the hyprland-nvidia-git package no longer works. I have managed to temporarily repair the installation. Since the hyprland-git package is now also recommended for Nvidia users, I installed hyprland-git manually after running the script and detecting an error during the installation of hyprland-nvidia-git. I then opened the install.sh script with Nano and replaced hyprland-nvidia-git with hyprland-git in line 94. Then I ran the script again and apparently everything worked.
